softwareComponents/isoreconfig/main.cpp

```diff
@@ -0,0 +1,27 @@
+#include <IO.h>
+#include <Configuration.h>
+#include "BFS.h"
+
+int main(int argc, char* argv[]) {
```

Comment:
Cosider using [Dim::Cli](https://gknowles.github.io/dimcli/v6.2.0/guide.html) for parsing the compile options. It will make your life easier.